<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReceiversWidgetClass</class>
 <widget class="QWidget" name="ReceiversWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>235</height>
   </rect>
  </property>
  <layout class="QVBoxLayout">
   <property name="margin">
    <number>2</number>
   </property>
   <item>
    <widget class="QToolButton" name="fastSelectButton">
     <property name="text">
      <string>Fast select</string>
     </property>
     <property name="popupMode">
      <enum>QToolButton::InstantPopup</enum>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextBesideIcon</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTreeWidget" name="trwReceivers">
     <property name="columnCount">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QPushButton" name="pbtAdd">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbtUpdate">
       <property name="text">
        <string>Update</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbtSelectAll">
       <property name="text">
        <string>All</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbtSelectNone">
       <property name="text">
        <string>None</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
